src/main/java/org/usfirst/frc/team4099/robot/Constants.kt

```diff
@@ -21,6 +35,18 @@ class Constants {
         val RIGHT_MASTER_ID = 4
         val RIGHT_SLAVE_1_ID = 5
         val RIGHT_SLAVE_2_ID = 0
+
+        val HIGH_GEAR_MAX_SETPOINT = 17.0 * 12.0 //17 fps
+    }
+
+    object Wheels {
+        val DRIVE_WHEEL_DIAMETER_INCHES=0
+    }
+
+    object Velocity {
+        val highGearVelocityControlSlot=0
```

Comment:
Same as above, also constants should be capitalized and underscore separated (there's a name for this but I can't remember what it is)